lib/bandwidth/web_rtc_lib/utils/web_rtc_transfer.rb

```diff
@@ -6,12 +6,12 @@
 
 module Bandwidth
   module WebRtc
-    def generate_bxml(device_token, sip_uri="sip:sipx.webrtc.bandwidth.com:5060")
-        return '<?xml version="1.0" encoding="UTF-8"?><Response>' + generate_transfer_bxml_verb(device_token, sip_uri) + '</Response>'
+    def generate_bxml(device_token, voice_call_id, sip_uri="sip:sipx.webrtc.bandwidth.com:5060")
+        return '<?xml version="1.0" encoding="UTF-8"?><Response>' + generate_transfer_bxml_verb(device_token, voice_call_id, sip_uri) + '</Response>'
     end
 
-    def generate_transfer_bxml_verb(device_token, sip_uri="sip:sipx.webrtc.bandwidth.com:5060")
-        return '<Transfer><SipUri uui="%s;encoding=jwt">%s</SipUri></Transfer>' % [device_token, sip_uri]
+    def generate_transfer_bxml_verb(device_token, voice_call_id, sip_uri="sip:sipx.webrtc.bandwidth.com:5060")
+        return '<Transfer><SipUri uui="%s;encoding=base64,%s;encoding=jwt">%s</SipUri></Transfer>' % [device_token, voice_call_id, sip_uri]
```

Comment:
Correct me if I'm wrong but I think voice_call_id needs to be the first parameter in the array at the end given the order of the %s values, i.e. `[voice_call_id, device_token, sip_uri]

Reply:
Yeah it's order dependent. Honestly we could go with named parameters to avoid any confusion

```
'Hello, %{first_name} %{last_name}!' % {first_name: 'John', last_name: 'Doe'}
```

https://stackoverflow.com/questions/23920404/named-format-string-parameters-with-format-specifiers/23920641

Reply:
Thank you! I'll update along with the proper string stripping.